src/pages/themes/index.astro

```diff
@@ -0,0 +1,189 @@
+	{
+		href: '/themes/1/?search=&categories%5B%5D=docs',
+		title: 'Documentation',
+		description: 'Ship your next technical documentation website with an Astro docs theme.',
+		cta: 'Browse docs themes',
+		image: Docs,
+	},
+	{
+		href: '/themes/1/',
+		title: '...and more!',
+		description:
+			'Discover all Astro themes and filter by your favorite technology, from Tailwind to Vue.',
+		cta: 'Browse all themes',
+		image: More,
+	},
+];
+
+const seo = {
+	title: 'Themes',
+	description: 'Elevate your style with our stellar designs.',
+};
+---
+
+<MainLayout {...seo}>
+	<div class="relative pb-16 overflow-hidden">
+		<div class="relative grid-container py-16 sm:py-20 md:py-24 xl:py-32">
+			<div class="absolute inset-0 bg-grid mask-linear-gradient-to-b bleed-full"></div>
+			<div class="relative flex flex-col items-center gap-6 text-center">
+				<h1 class="mx-auto max-w-screen-lg heading-3 sm:heading-2 md:heading-1">
+					Jumpstart your next project with a theme
+				</h1>
+			</div>
+		</div>
+
+		<div class="grid-container pt-16 border-astro-gray-500 border-t overflow-hidden">
+			<section class="space-y-14">
+				<div>
+					<h2 class="heading-4 sm:heading-3 md:heading-2" id="featured">Featured themes</h2>
+					<p class="text-astro-gray-200 body md:body-large">
+						Professional themes designed and developed by our sponsors
+					</p>
+				</div>
+				<div class="gap-6 grid md:grid-cols-2 lg:grid-cols-3">
+					{featured.map((theme) => <ThemeCard {theme} />)}
+				</div>
+				<div class="flex justify-center">
+					<a href="/themes/1/" class="button button-primary"> Discover more themes </a>
+				</div>
+			</section>
+
+			<section class="space-y-14">
+				<div>
+					<span class="bg-red-pink-gradient bg-clip-text font-semibold text-transparent"
+						>Find the theme for you</span
+					>
+
+					<h2
+						class="mt-1 text-white text-balance heading-4 sm:heading-3 md:heading-2"
+						id="categories"
+					>
+						Launch your next site at light speed with our catalog of free and premium Astro themes
+					</h2>
+				</div>
+
+				<div class="md:max-lg:hidden gap-2 lg:gap-4 grid grid-cols-1 lg:grid-cols-3 mt-4 sm:mt-8">
+					{
+						categories.map(({ href, title, description, cta, image }) => (
+							<a
+								{href}
+								class="group flex flex-col p-1 border border-astro-dark-100/20 hover:border-astro-dark-100/40 rounded-xl xl:rounded-2xl overflow-hidden"
+							>
+								<div class="space-y-2 p-3">
+									<h3 class="font-semibold text-white text-base">{title}</h3>
+									<p class="hidden sm:block font-light text-astro-gray-200 text-sm xl:text-base md:text-balance">
+										{description}
+									</p>
+								</div>
+								<div class="relative">
+									<Image
+										src={image}
+										alt=""
+										class="rounded-lg xl:rounded-xl w-full h-48 object-cover"
+									/>
+									<div class="absolute inset-0 flex justify-center items-center opacity-0 group-hover:opacity-100 transition-opacity">
```

Comment:
I think the hover styles look pretty good! Just a few notes:

- I think we need to dim the screenshots slightly with a dark overlay when displaying the “Browse…” call to action as contrast isn’t great currently (something like the theme cards on the homepage for example).
- We should make sure these hover styles also work when someone focuses the link. (Given we should move the link to the heading, this will likely need something like a group on the card, and `group-focus-within` or whatever it is in Tailwind?)
- I think it would be nice to always show the “Browse all themes” CTA on the last card, otherwise it feels a little empty.